### PR TITLE
[IMP] theme_*: adapt `s_references_grid` to themes

### PR DESCRIPTION
**Pull request by mano-odoo**
* anelusia, buzzy, kea, nano, odoo_experts, zap

Adapt the newly introduced `s_references_grid` to match existing themes.

task-4146798
Part-of: task-4077427

Community PR: https://github.com/odoo/odoo/pull/178359

### CODE REVIEW

File: theme_nano/views/snippets/s_references_grid.xml
Comment (anso-odoo) [inline theme_nano/views/snippets/s_references_grid.xml:1]: Not sure about the contrast here
Screenshot 2024-09-06 at 17 32 51

Also, I the logos are cut, we should increase their grid height by 1
Screenshot 2024-09-06 at 17 35 02

**Review by anso-odoo — CHANGES_REQUESTED** [review]
Hey @mano-odoo, just some small issues 👍 

---

*Formatting nitpick mode ON*

Don't forget to codify ("`blabla`") the snippet title in the commit and PR title
Also if you use `theme_*` in the title,  it should be "*: anelusia, artists,..." in the message

*Formatting nitpick mode OFF*

File: theme_nano/views/snippets/s_references_grid.xml
Comment (mano-odoo) [reply theme_nano/views/snippets/s_references_grid.xml:1]: As discussed visual result now: 
image

**Review by anso-odoo — APPROVED** [review]

File: theme_zap/views/snippets/s_references_grid.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc2" remove="o_cc1" separator=" "/>
+    </xpath>
+    <!-- Grid gap -->
+    <xpath expr="//div[hasclass('o_grid_mode')]" position="attributes">
+        <attribute name="style" add="gap: 8px;" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        Our <b>References</b>
```
Comment (bvr-odoo) [inline theme_zap/views/snippets/s_references_grid.xml:15]: Small change here: The editor cannot add `<b>` tags. It adds `<strong>` tags for bold :wink:

**Review by bvr-odoo — CHANGES_REQUESTED** [review]

**Review by bvr-odoo — APPROVED** [review]

**Comment by SergeBayet** [comment]
Commit(s) moved into
- https://github.com/odoo/design-themes/pull/919

**Pull request closed without merging**